research/auto_fed_rl/index.html

```diff
@@ -0,0 +1,16 @@
+<html xmlns="http://www.w3.org/1999/xhtml">
+<head>
+    <title>dints</title>
+    <meta http-equiv="refresh" content="0;URL='https://github.com/NVIDIA/NVFlare/blob/dev/research/auto_fed_rl'" />
+</head>
+<body>
+<p>
+    Redirecting to the project codebase:
+</p>
+<p>
+    <a href="https://github.com/NVIDIA/NVFlare/blob/dev/research/auto_fed_rl">
+        https://github.com/NVIDIA/NVFlare/blob/dev/research/auto_fed_rl
+    </a>
+</p>
+</body>
+</html>
```

Comment:
Our documentation is build automatically in the "doc" branch.
@nvkevlu @IsaacYangSLA if we want to add the re-direction, should we PR into that branch?
Can you advise?
Thanks

Reply:
Got it,
The title should be changed to something other than "dints"